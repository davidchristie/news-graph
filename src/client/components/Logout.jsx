import React, { Component, PropTypes } from 'react'

export default class Logout extends Component {
  render () {
    const onClick = this.props.onClick
    return (
      <button className='btn btn-danger' onClick={onClick}>
        Logout
      </button>
    )
  }
}

Logout.propTypes = {
  onClick: PropTypes.func.isRequired
}
