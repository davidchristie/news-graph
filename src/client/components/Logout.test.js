/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Logout from './Logout'

const onClick = () => {}

test('Logout renders correctly', () => {
  const wrapper = shallow(<Logout onClick={onClick} />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
