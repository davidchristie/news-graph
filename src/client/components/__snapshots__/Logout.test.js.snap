exports[`test Logout renders correctly 1`] = `
<button
  className="btn btn-danger"
  onClick={[Function]}>
  Logout
</button>
`;
